Felienne/hedy
# [FEATURE] Enables teachers to create their own adventures

**Description**
In this PR we implement a new feature enabling teachers to create, edit and delete their own adventures! We add a new section to the `for-teachers` page called "My adventures". Teachers can create a new adventure and are able to select a corresponding Hedy level and use commands and code snippets as well. Created adventures can be implemented in the adventures tab of classes under "class customizations".

**Fix for**
This PR fixes #1515.

**How to test**
Due to the complexity and large amount of related files this need quite some testing! Make sure you are a teacher and navigate to the teachers account and notice the new section. Create an adventure, alter it, delete it or select any other option. Then make sure you have a class and navigate to class customizations. Notice that the created level is now visible as an option in the corresponding Hedy level. Select it, login as a student (or join your own class) and navigate to that specific level. Notice that the adventure can be found within the adventure tab in a distinct color.

**Screenshots**
![Schermafbeelding 2022-01-27 om 15 00 57](https://user-images.githubusercontent.com/38155424/151373577-8337ff64-8d78-4269-b628-9dcb1d6f10b8.png)
![Schermafbeelding 2022-01-27 om 15 00 49](https://user-images.githubusercontent.com/38155424/151373596-8ca22b70-abd7-41d0-966a-8d13eddbb881.png)
![Schermafbeelding 2022-01-27 om 16 55 16](https://user-images.githubusercontent.com/38155424/151394966-7eb62ebc-a153-4a5f-b73f-2f1ed17a9df8.png)
![Schermafbeelding 2022-01-27 om 16 55 35](https://user-images.githubusercontent.com/38155424/151394985-97dc2e46-8b5d-41de-947e-fa8b5b1d0fce.png)
![Schermafbeelding 2022-01-27 om 16 55 49](https://user-images.githubusercontent.com/38155424/151395007-8677d885-1948-4112-817b-572c3f584cd7.png)
